### DISCUSSION TITLE
[1126320] Being a DIK [本体]

### DISCUSSION THREAD

**Discussion by liusj5257** [pjy612/SteamManifestCache#127] [Share]
`appid：1126320`

keys：
```
1126320;d8008bebfc5dcd66a109007d2238b7074a3417874a199e919c05d9a91711668a
1126321;b487bb317f141b1ea8c7f43473ae7192dc9c3ddabf47e75c0b8ea091ea83f5dd
```

[depotcache.zip](https://github.com/pjy612/SteamManifestCache/files/13987866/depotcache.zip)

内测码：2E3B329F736C3085

邮箱：[574850302@qq.com](mailto:574850302@qq.com)

**Comment by pjy612** [comment]
密钥配置有误，或不全
https://steamdb.info/app/1126320/depots/

**Reply by liusj5257** [reply]
我看了在Steam\config\config.vdf里面并没有搜到1126321，我电脑已经安装过并玩过这个游戏的，要怎么找到它？

编辑找到了，在另一个磁盘的steam库里也有一个config.vdf文件的里面

**Reply by pjy612** [reply]
查收

**Reply by liusj5257** [reply]
@pjy612 刚邮箱联系了,麻烦重发下秘钥 2E3B329F736C3085

**Reply by pjy612** [reply]
查收邀请
